content/en/seachat/seachat-manual/03-add-knowledge/09-add-webpage-link-in-answers.md

```diff
@@ -92,18 +92,28 @@ Once there is information in the knowledge base, the agent will start using this
     <p style="margin-top: 20px; font-size: 15px">URLs added to Agent's Answer</p>
 </div>
 
-### KB IDs
+### Limits of Button Messages
 
 <div id="additional-setting-ui" style="display: flex; flex-direction: column; align-items: center;">
 <div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
-    <a href="/images/seachat/en/knowledge-advanced-features/url-button/kb-id.png" target="_blank">
-    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/knowledge-advanced-features/url-button/kb-id.png" alt="">
+    <a href="/images/seachat/en/knowledge-advanced-features/url-button/kb-id-problem.png" target="_blank">
+    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/knowledge-advanced-features/url-button/kb-id-problem.png" alt="">
     </a>
 </div>
+    <p style="margin-top: 20px; font-size: 15px">Message Cutoff due to Message Limitation</p>
 </div>
-
 <br/>
 
+Here is a summary of the current limits for our button templates and postback buttons:
```

Comment:
this is still not clear to me: is the limit from webchat or Line?

Originally when we introduced this, it's supposed to resolve the Line limitation. But here it doens't mention Line at al!!!

Reply:
I will move this limitation to the section of Line and reference it from here.